chromium/pages/cancel/index.html

```diff
@@ -8,11 +8,18 @@
     <link rel="stylesheet" href="style.css" type="text/css" />
   </head>
   <body>
-    <h1 id="https-everywhere"><img src="../../images/banner-red.png" alt="HTTPS Everywhere" /></h1>
-    <p data-i18n="cancel_he_blocking_explainer"></p>
-    <p id="url-paragraph"><span id="url-label">URL: </span><a href="#" id="originURL"></a></p>
-    <script src="../translation.js"></script>
-    <script src="../util.js"></script>
-    <script src="ux.js"></script>
+    <div class="content">
+      <h1 id="https-everywhere"><img src="../../images/banner-red.png" alt="HTTPS Everywhere" /></h1>
+      <p data-i18n="cancel_he_blocking_explainer"></p>
+      <p id="url-paragraph">
+        <span id="url-label">URL: </span><a href="#" id="url-value">PLACEHOLDER</a>
```

Comment:
make this a `<span>` that autoselects if you click it.

Reply:
I think it's valuable to know whether you've visited the URL before. A user may remember that they visited a website a few years ago but not quite remember the exact URL. If they search for it on http://excite.com/ and click an imposter link the :visited status can expose that.

Reply:
@patch-malone You need to make the link show a message box that asks if it's ok to visit unencrypted website then.

Reply:
I see. Do you think the <span> is better?

Reply:
@patch-malone Yes.